### ISSUE TITLE
link definition title on line following link url is parsed as a separate paragraph

### ISSUE THREAD

**Issue by grant-g** [chjj/marked#498]
http://daringfireball.net/projects/markdown/syntax#link says:
'
You can put the title attribute on the next line and use extra spaces or tabs for padding, which tends to look better with longer URLs:

```
[id]: http://example.com/longish/path/to/resource/here
    "Optional Title Here"
```

'

However when marked sees this it parses the title line as a separate paragraph from the definition on the line above.

**Comment by joshbruce** [comment]
#984

**Comment by Feder1co5oave** [comment]
This was fixed by #1018:

```markdown
[id]: http://example.com/longish/path/to/resource/here
    "Optional Title Here"

[id]
```
~~~html
<p><a href="http://example.com/longish/path/to/resource/here" title="Optional Title Here">id</a></p>
~~~

@joshbruce
